### ISSUE TITLE
Mobile version does not work (iOS only)

### ISSUE THREAD

**Issue by neophob** [neophob/wpc-emu#13]
the wpc emu does not work anymore on mobile browser

error: `SyntaxError: Cannot declare a let variable twice: 'e'.` (Safari on iOS)

**Comment by neophob** [comment]
switched to terser optimiser, now `NotSupportedError (DOM Exception 9): The operation is not supported.` on ios

**Comment by neophob** [comment]
Looks like the source of the issue is here:

```
this.buffer = this.audioCtx.createBuffer(MONO, DEFAULT_SAMPLE_SIZE, INPUT_SAMPLE_RATE_HZ),
```

**Comment by neophob** [comment]
fixed in v0.6.10
